src/test/java/uk/sky/cqlmigrate/example/CmdLineEntryPoint.java

```diff
@@ -0,0 +1,32 @@
+package uk.sky.cqlmigrate.example;
+
+public class CmdLineEntryPoint {
+
+    public static void main(String[] args) throws Exception {
+
+        int returnCode = 1;
+
+        try {
+            CqlMigrateInvoker cqlMigrateInvoker = new CqlMigrateInvoker();
+
+            // @Before
+            CqlMigrateInvoker.setupCassandra();
+            cqlMigrateInvoker.setUp();
+
+            // Do the test
+            cqlMigrateInvoker.doMigrate();
+
+            // @After
+            cqlMigrateInvoker.tearDown();
+            CqlMigrateInvoker.tearDownCassandra();
+
+            returnCode = 0;
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
+        finally {
+            System.exit(returnCode);
```

Comment:
This is subjective but in general I prefer to avoid mutable variables even if it adds code duplication. I'd prefer an explicit System.exit(0) at the end of the try and System.exit(1) in the catch. WDYT @jsravn ?